docker/package/packages.py

```diff
@@ -71,7 +71,7 @@
 
 node_units = []
 for network in networks:
-    env = [f"DATA_DIR=/var/lib/tezos/node-{network}", f"NETWORK={network}", "NODE_RPC_PORT=8732",
+    env = [f"DATA_DIR=/var/lib/tezos/node-{network}", f"NETWORK={network}", "NODE_RPC_PORT=:8732",
```

Comment:
Is `NODE_RPC_PORT` still used? Also did you grep `NODE_RPC_PORT` and `NODE_HOST` to make sure they are no longer mentioned in all places where should stop being mentioned?

Reply:
This is actually a part of `tezos-node` systemd service, where the node rpc port should be specified a bit different via `--rpc-addr=ADDR:PORT` option. However, it's indeed weird that only port can be specified

> Also did you grep NODE_RPC_PORT and NODE_HOST to make sure they are no longer mentioned in all places where should stop being mentioned?

Yes